client/my-sites/email/titan-setup-thank-you/index.tsx

```diff
@@ -0,0 +1,138 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { ThemeProvider } from 'emotion-theming';
+import { connect, useSelector } from 'react-redux';
+import { localize, useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { ThankYou } from 'calypso/components/thank-you';
+import theme from 'calypso/my-sites/marketplace/theme';
+import { FullWidthButton } from 'calypso/my-sites/marketplace/components';
+import {
+	emailManagement,
+	emailManagementTitanControlPanelRedirect,
+} from 'calypso/my-sites/email/paths';
+import { getSelectedSite, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import { getTitanEmailUrl } from 'calypso/lib/titan';
+import Gridicon from 'calypso/components/gridicon';
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
+import { getSelectedDomain } from 'calypso/lib/domains';
+import { getDomainsBySite } from 'calypso/state/sites/domains/selectors';
+import {
+	getEmailPurchaseByDomain,
+	hasEmailSubscription,
+} from 'calypso/my-sites/email/email-management/home/utils';
+import {
+	hasLoadedSitePurchasesFromServer,
+	isFetchingSitePurchases,
+} from 'calypso/state/purchases/selectors';
+import { SiteData } from 'calypso/state/ui/selectors/site-data';
+import { TITAN_CONTROL_PANEL_CONTEXT_GET_MOBILE_APP } from 'calypso/lib/titan/constants';
+
+/**
+ * Import styles
+ */
+import './style.scss';
+
+type TitanSetupThankYouProps = {
+	domainName: string;
+	emailAddress: string;
+};
+
+const TitanSetupThankYou = ( props: TitanSetupThankYouProps ): JSX.Element => {
+	const currentRoute = useSelector( getCurrentRoute );
+	const selectedSiteSlug = useSelector( getSelectedSiteSlug );
+	const translate = useTranslate();
+
+	const { domainName, emailAddress } = props;
+
+	const emailManagementPath = emailManagement( selectedSiteSlug, domainName, currentRoute );
+
+	const thankYouImage = {
+		alt: translate( 'Thank you' ),
+		src: '/calypso/images/upgrades/thank-you.svg',
+	};
+
+	const getControlPanelUrl = ( context: string ) => {
+		return emailManagementTitanControlPanelRedirect( selectedSiteSlug, domainName, currentRoute, {
+			context,
+		} );
+	};
```

Comment:
Nit: It looks like we're only using this function from one location. So why not just set the redirect URL directly into a constant?
```suggestion
	const titanControlPanelUrl = emailManagementTitanControlPanelRedirect(
		selectedSiteSlug,
		domainName,
		currentRoute, {
			context: TITAN_CONTROL_PANEL_CONTEXT_GET_MOBILE_APP,
		}
	);
```